web/src/hash_util.ts

```diff
@@ -198,7 +198,7 @@ export function channels_settings_edit_url(
 }
 
 export function channels_settings_section_url(section = "subscribed"): string {
-    const valid_section_values = new Set(["new", "subscribed", "all"]);
+    const valid_section_values = new Set(["new", "subscribed", "all", "notsubscribed"]);
```

Comment:
This change should be in the commit which adds the new hash, i.e. which adds the new tab.